_data-prepper/pipelines/configuration/processors/list-to-map.md

```diff
@@ -16,10 +16,12 @@ The following table describes the configuration options used to generate target
 
 Option | Required | Type | Description
 :--- | :--- | :--- | :---
-`key` | Yes | String | The key of the fields to be extracted as keys in the generated mappings.
 `source` | Yes | String | The list of objects with `key` fields to be converted into keys for the generated map.
 `target` | No | String | The target for the generated map. When not specified, the generated map will be placed in the root node.
+`key` | No | String | The key of the fields to be extracted as keys in the generated mappings. Must be specified if `use_source_key` is `false`.
```

Comment:
For the required (second) column, this has "No". But, some of our documents state "Conditionally" when the value is sometimes required. Maybe we should change that.